package/MDAnalysis/coordinates/PDB.py

```diff
@@ -275,27 +275,16 @@ def __init__(self, filename, **kwargs):
         pdbfile = self._pdbfile = util.anyopen(filename)
 
         line = "magical"
-        # we handle windows file striding uniquely
-        # because file.tell() is not reliable on Windows
-        # for a text file and binary mode is not compatible
-        # with i.e., our use of anyopen for zip files
-        windows_overbyte = 0
         while line:
             # need to use readline so tell gives end of line
             # (rather than end of current chunk)
             line = pdbfile.readline()
-            # the tell() text opaque number striding is not portable
-            # to windows so compensate accordingly
-            if os.name == 'nt':
-                # for a text file mode these aren't necessarily
-                # bytes; var name for simplicity only
-                windows_overbyte += 1
```

Comment:
doesn't this break files we wrote with windows? @tylerjereddy do you remember why you included those lines?

Reply:
Yeah this accounted for the extra newline char.  This reverts @tylerjereddy 's fix which fixed Windows-written files but broke Unix-written files (which includes the PDB itself).  [See here](https://github.com/MDAnalysis/mdanalysis/issues/2128#issuecomment-434490576) but I think there's a universal fix possible, this patch just reverts this change so that a 0.19.1 can read most(?) PDB files

Reply:
The Python 2.7.4 [standard docs for `file.tell()`](https://docs.python.org/2/library/stdtypes.html#file.tell) indicate:

> On Windows, `tell()` can return illegal values (after an fgets()) when reading files with Unix-style line-endings. Use binary mode ('rb') to circumvent this problem.

If we didn't use `anyopen` my impression is that we could fix the problem.

My comment indicating that the results are **opaque** numbers and not necessarily the number of bytes since the start of the file is discussed in some detail in [a SO post](https://stackoverflow.com/a/15935038/2942522).